src/Node.jsx

```diff
@@ -42,7 +42,7 @@ class Node extends React.Component {
   }
 
   famousDelete() {
-    this._famousNode.dismount();
+    this._famousNode.getParent().removeChild(this._famousNode);
```

Comment:
@pilwon This throws if the node has no parent. Maybe we should check if the node has a parent first?